cli/cmd/launch.py

```diff
@@ -51,15 +51,13 @@ def _launch(config, cookies, sys_uuid, vios_uuids):
         active_vios_servers = vios_operation.get_active_vios(
             config, cookies, sys_uuid, vios_uuids)
         if len(active_vios_servers) == 0:
-            logger.error("failed to find active VIOS server")
-            raise VIOSError("failed to find active VIOS server")
+            raise VIOSError(f"no active VIOS server attached to the system '{util.get_host_address(config)}'")
```

Comment:
host address will be pointing to HMC IP. probably we can mention the system id or get correspondign system name and log.